python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -23,6 +23,7 @@
 import sgtk
 from sgtk.commands.clone_configuration import clone_pipeline_configuration_html
 from sgtk.util import process
+from . import constants
```

Comment:
I'll let you and @manneohrstrom duke that out. I moved things to constants.py after an earlier CR command from him. I'm not passionate enough about one approach over the other to make an argument for either.